plugins/container_logs.yaml

```diff
@@ -50,12 +50,20 @@ parameters:
       # This format is not well documented: https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/kuberuntime/logs/logs.go#L125-L169
       - containerd-cri
     default: auto
-
+  - name: offset_storage_dir
+    description: The directory that the offset storage file will be created
+    type: string
+    default: $OIQ_OTEL_COLLECTOR_STORAGE
```

Comment:
I think the supplied_plugins_test is choking because this isn't `$OIQ_OTEL_COLLECTOR_HOME/storage`; 

I do think `$OIQ_OTEL_COLLECTOR_STORAGE` is the correct variable to use, but looks like everything else is using `$OIQ_OTEL_COLLECTOR_HOME/storage`, so we should probably use that just to stay in-line with the other plugins.